src/widget/wsearchlineedit.cpp

```diff
@@ -614,7 +638,7 @@ void WSearchLineEdit::updateClearAndDropdownButton(const QString& text) {
     const int paddingPx = text.isEmpty() ? 0 : m_innerHeight;
     const QString clearPos(layoutDirection() == Qt::RightToLeft ? "left" : "right");
 
-    // Hide the nonfunctional drop-down button if the search is disabled.
+    // 'Hide' the nonfunctional drop-down button if the search is disabled.
```

Comment:
Hide ... by ...???. Please remove the confusing quotes and instead provide an explanation if needed.

Reply:
done.